src/renderer/extensions/vueNodes/widgets/components/WidgetToggleSwitch.vue

```diff
@@ -21,13 +30,25 @@ import {
 
 import WidgetLayoutField from './layout/WidgetLayoutField.vue'
 
+interface BooleanWidgetOptions {
+  on?: string
+  off?: string
+  [key: string]: any
+}
+
```

Comment:
When would it be both on and off?
Also, why the `any` index signature?

Reply:
_When would it be both on and off?_ - Both labels are typically provided together (or neither). The options come from `label_on` and `label_off `( nodeDefSchema.ts ) in the input spec (useBooleanWidget.ts). When only one is provided, it is  defaulting the missing one (labelOn defaults to 'true', labelOff to 'false'). 
_Also, why the any index signature?_ - The index signature (now `unknown`, not `any` - that was my mistake!) is needed because widget.options can contain additional properties beyond on/off that get filtered and passed through to the underlying component via filterWidgetProps.